src/test/java/com/vinylteam/vinyl/dao/jdbc/JdbcOfferDaoITest.java

```diff
@@ -1,3 +1,5 @@
+
+/*
 package com.vinylteam.vinyl.dao.jdbc;
```

Comment:
We hace to figure out what to do with commented classes like this one.

Reply:
i think can be deleted for Offer and UniqueVinyl